CRM/Grant/Form/Search.php

```diff
@@ -86,7 +88,7 @@ public function preProcess() {
       $this->_context
     );
     $prefix = NULL;
-    if ($this->_context == 'user') {
+    if ($this->_context === 'user') {
```

Comment:
@eileenmcnaughton any specific reason for this?

Reply:
@yashodha yes - using strict comparison is better coding practice. We should use it wherever we can. In this case we know we can because only a string could match 'user'